exercises/say/SayTest.cs

```diff
@@ -1,101 +1,96 @@
     }
 
     [Fact(Skip = "Remove to run test")]
     public void Twenty()
     {
-        Assert.Equal("twenty", Say.InEnglish(20L));
+        Assert.Equal("twenty", Say.InEnglish(20));
     }
 
     [Fact(Skip = "Remove to run test")]
     public void Twenty_two()
     {
-        Assert.Equal("twenty-two", Say.InEnglish(22L));
+        Assert.Equal("twenty-two", Say.InEnglish(22));
     }
 
     [Fact(Skip = "Remove to run test")]
     public void One_hundred()
     {
-        Assert.Equal("one hundred", Say.InEnglish(100L));
+        Assert.Equal("one hundred", Say.InEnglish(100));
     }
 
     [Fact(Skip = "Remove to run test")]
     public void One_hundred_twenty_three()
     {
-        Assert.Equal("one hundred twenty-three", Say.InEnglish(123L));
+        Assert.Equal("one hundred twenty-three", Say.InEnglish(123));
     }
 
     [Fact(Skip = "Remove to run test")]
     public void One_thousand()
     {
-        Assert.Equal("one thousand", Say.InEnglish(1000L));
+        Assert.Equal("one thousand", Say.InEnglish(1000));
     }
 
     [Fact(Skip = "Remove to run test")]
     public void One_thousand_two_hundred_thirty_four()
     {
-        Assert.Equal("one thousand two hundred thirty-four", Say.InEnglish(1234L));
+        Assert.Equal("one thousand two hundred thirty-four", Say.InEnglish(1234));
     }
 
     [Fact(Skip = "Remove to run test")]
     public void One_million()
     {
-        Assert.Equal("one million", Say.InEnglish(1000000L));
-    }
-
-    [Fact(Skip = "Remove to run test")]
-    public void One_million_two()
-    {
-        Assert.Equal("one million two", Say.InEnglish(1000002L));
+        Assert.Equal("one million", Say.InEnglish(1000000));
     }
 
     [Fact(Skip = "Remove to run test")]
     public void One_million_two_thousand_three_hundred_forty_five()
     {
-        Assert.Equal("one million two thousand three hundred forty-five", Say.InEnglish(1002345L));
+        Assert.Equal("one million two thousand three hundred forty-five", Say.InEnglish(1002345));
     }
 
     [Fact(Skip = "Remove to run test")]
     public void One_billion()
     {
-        Assert.Equal("one billion", Say.InEnglish(1000000000L));
+        Assert.Equal("one billion", Say.InEnglish(1000000000));
     }
 
     [Fact(Skip = "Remove to run test")]
     public void A_big_number()
     {
-        Assert.Equal("nine hundred eighty-seven billion six hundred fifty-four million three hundred twenty-one thousand one hundred twenty-three", Say.InEnglish(987654321123L));
+        Assert.Equal("nine hundred eighty-seven billion six hundred fifty-four million three hundred twenty-one thousand one hundred twenty-three", Say.InEnglish(987654321123));
     }
 
     [Fact(Skip = "Remove to run test")]
-    public void Lower_bound()
+    public void Numbers_below_zero_are_out_of_range()
     {
-        Assert.Throws<ArgumentOutOfRangeException>(() => Say.InEnglish(-1L));
+        Assert.Equal("-1", Say.InEnglish(-1));
```

Comment:
I don't really like the change to -1 instead of throwing an exception. Was that on purpose or just to conform to the canonical data? If it is the latter, you should know that some canonical data exercises use -1 to signal errors, some use `null` and other use an `error` object. In this case, I think we should keep the exception throwing.